lib/winston/transports/file.js

```diff
@@ -558,10 +558,13 @@ File.prototype._createStream = function () {
   })(this._getFile());
 };
 
+File.prototype._getFilename = function () {
+  return this._basename;
+};
```

Comment:
This addition is not strictly breaking, but it is now being used as a substitute for `this.filename`, the removal of which is breaking (see: https://github.com/winstonjs/winston/pull/829/files/62a111d0e8a3b83509fe693ecb6eb48ae6bb0042#r63998609)